datafusion/spark/src/function/bitwise/bit_count.rs

```diff
@@ -147,6 +147,18 @@ fn spark_bit_count(value_array: &[ArrayRef]) -> Result<ArrayRef> {
     }
 }
 
+// Here’s the equivalent Rust implementation of the bitCount function (similar to Apache Spark's bitCount for LongType)
+fn bit_count(i: i64) -> i32 {
```

Comment:
https://github.com/apache/spark/blob/7ac2b5423362e1040f4c5c63b0d4ba6fc999a465/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/bitwiseExpressions.scala#L243

Reply:
Thanks, added

Reply:
Where was the actual bitcount implementation code pulled from though?